Tests/SwiftDriverTests/IncrementalCompilationTests.swift

```
@@ -136,9 +136,9 @@ extension IncrementalCompilationTests {
   /// Much of the code below is taking from testLinking(), but uses the output file map code here.
   func testAutolinkOutputPath() {
 #if os(Windows)
-    XCTSkip("Driver.init fatalError")
+    // XCTSkip("Driver.init fatalError")
     return
```

`throw XCTSkip` would be better than silently returning.

---

I agree ... except the test isn't marked as throwing, so I cannot throw in the context.  If that is preferred, I can mark the test as `throws`.  I don't have an issue with that.

---

I think changing it to `throws` is fine. The error is handled by XCTest.